src/LaunchDarkly.Client/IStoreEvents.cs

```diff
@@ -1,6 +1,8 @@
-﻿namespace LaunchDarkly.Client
+﻿using System;
+
+namespace LaunchDarkly.Client
 {
-    public interface IStoreEvents
+    public interface IStoreEvents : IDisposable
```

Comment:
This is a breaking interface change, but a good one.  Anyone implementing IStoreEvents, and sending events remotely, is going to have a disposable object.

Reply:
If they're sending events remotely, yes, but not necessarily if it's a test stub or a file appender, etc. I agree that we should be calling `Dispose` on anything that really is disposable as mentioned [here](https://github.com/launchdarkly/.net-client/pull/70#discussion-diff-170115900L261), but I'm not sure that requires this breaking change.